CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -37,6 +58,69 @@ export default function Report({ reportID }) {
             <h3>Number of Cases Different: {results.length}</h3>
           </div>
 
+		  {/* Report and Disease Specific Stats */}
+		  <div>
+			{statistics && (
+			<>
+			<button
+				type="button"
+				className="bg-blue-400 text-white px-5 py-2 rounded-md hover:bg-blue-600 mb-4"
+				onClick={toggleDiseaseStats}
+			>
+				{showDiseaseSpecificStats ? "Hide" : "Show"} Disease Specific Stats
+			</button>
+
+			{!showDiseaseSpecificStats ? (
```

Comment:
showDiseaseSpecificStats is not a defined variable. I think you meant to use showDiseaseStats